### PR TITLE
Clarifications on order of assignment

### PR DESCRIPTION
**Pull request by markkramerus**

### CODE REVIEW

File: input/pagecontent/reference.md [hunk 1]
```diff
@@ -667,9 +669,13 @@ In the following, we give details and examples of assignments involving various
...
 
-FSH provides a shortcut for setting several of the most common properties of a Coding simultaneously, namely, system, version, code, and display. The syntax is:
+A FHIR Coding has five attributes (system, version, code, display, and userSelected). The first four of these can be set with a single assignment statement. The syntax is:
+
+<pre><code>&lt;Coding&gt; = <i>{CodeSystem name|id|url}|{version string}</i>#{code} <i>"{display string}"</i></code></pre>
 
-<pre><code>&lt;Coding&gt; = {CodeSystem name|id|url}<i>|{version string}</i>#{code} <i>"{display string}"</i></code></pre>
+The only required part of this statement is the code (including the # sign), although it is rare to have a Coding without a code system. The version string cannot appear without a code system.
+
+Whenever this type of rule is applied, whatever is on the right side **entirely replaces** the previous value of the Coding on the left side. For example, if a Coding has a value that includes a display string, and a new assignment is made that replaces the system and code but has no display string, then result is a Coding without a display string.
```
Comment (ngfreiter) [inline input/pagecontent/reference.md:678]: typo: "**then** result is a Coding without a display string"

File: input/pagecontent/reference.md [hunk 2]
```diff
@@ -697,52 +703,127 @@ FSH provides a shortcut for setting several of the most common properties of a C
...
+
+  ```
+  * myCoding = SCT#363346000 "Malignant neoplastic disease (disorder)"
+  * myCoding = ICD10#C80.1
+  
+  // Result:
+  //   myCoding.system is ICD10
+  //   myCoding.code is C80.1
+  //   myCoding.display does not exist
+  //   myCoding.version does not exist
```
Comment (ngfreiter) [inline input/pagecontent/reference.md:716]: I'm probably just being picky so feel free to ignore me, but "does not exist" felt odd to me. I think something like "has no value" or "is not set to a value" feels more descriptive.

File: input/pagecontent/reference.md [hunk 3]
```diff
@@ -757,16 +838,16 @@ For other code systems, the value and units can also be set independently. To as
...
-* &lt;Quantity&gt; from {ValueSet name|id|url}
-</code></pre>
+A Quantity can also be bound to a value set:
 
-In the first expression, `CodeSystem` corresponds to Quantity.system, `code` to Quantity.code, and `display string` to Quantity.unit.
+<pre><code>* &lt;Quantity&gt; from {ValueSet name|id|url}
+</code></pre>
 
-> **Note:** The ability to assign a coded value or bind a value set directly to a Quantity is a consequence of FHIR's definition of Quantity as a coded data type, rather than using a Coding to represent the units of measure.
+> **Note:** The ability to assign a coded value or bind a value set directly to a Quantity is a consequence of FHIR's definition of Quantity as a coded data type (probably not the greatest idea in the world).
```
Comment (ngfreiter) [inline input/pagecontent/reference.md:850]: Woah this line isn't holding back on FHIR.

**Review by ngfreiter — COMMENTED** [review]
Looked mostly good to me. I had some minor comments.

File: input/pagecontent/reference.md (hunk 3 above)
Comment (markkramerus) [reply input/pagecontent/reference.md:850]: Ok, you're right.  "The ability to assign a coded value or bind a value set directly to a Quantity is a consequence of FHIR oddly treating Quantity as a coded data type."

File: input/pagecontent/reference.md (hunk 2 above)
Comment (markkramerus) [reply input/pagecontent/reference.md:716]: changed to "has no value"

File: input/pagecontent/reference.md (hunk 1 above)
Comment (markkramerus) [reply input/pagecontent/reference.md:678]: fixed

**Pull request merged**